gapis/trace/android/mali/validate.go

```diff
@@ -16,28 +16,44 @@ package mali
 
 import (
 	"context"
+	"strings"
 
 	"github.com/google/gapid/gapis/perfetto"
 	"github.com/google/gapid/gapis/trace/android/validate"
 )
 
 var (
-	// All counters must be inside this array.
-	counters = []validate.GpuCounter{
+	// Hardware Counters for JM GPUs
```

Comment:
Thanks for the review.

> For go, comments should start with the name of the thing they are documenting. From https://golang.org/doc/effective_go#commentary:

I changed the comments in complete sentences.

> So, please phrase it like // jmCounters are .... Please also refrain from using abbreviations in the documentation, since not many people reading this will be familiar with "JM" or "CSF".
> So, I think this one should be // jmCounters are the hardware counters found on job manager based GPUs.
> Please also fix the one for csfCounters.

Unfortunately our rules don't allow us to use anything other than the acronyms in the source code so I just left them as JM and CSF.